docs/writing-rules/autofix.md

```diff
@@ -24,6 +24,12 @@ rules:
   severity: WARNING
 ```
 
+## Creating autofix rules
+
+See creation of an autofix rule in Transforming code with Semgrep autofixes video:
```

Comment:
```suggestion
See how to create an autofix rule in **Transforming code with Semgrep autofixes** video:
```
I added the bold to make it clear that the noun phrase is a title.

Reply:
Good idea!